### PR TITLE
fix(theme): remove hard-coded light mode colors in favor of bundled ones

### PR DESCRIPTION
**Pull request by liamdebeasi**
We're now bundling the light mode colors directly with Ionic, so we can remove the hard-coded tokens to ensure the app receives updates. The dev build includes the latest AA contrast work.

### CODE REVIEW

File: src/global/app.css [hunk 1]
```diff
@@ -77,4 +68,9 @@ html {
...
 :root .safari_only ion-header .header-background {
   backdrop-filter: none !important;
-}
\ No newline at end of file
+}
+
+:root.ios {
+  --ion-padding: 20px;
+  --ion-margin: 20px;
+}
```
Comment (liamdebeasi) [inline src/global/app.css:76]: This was part of the now deleted `default.css` but does not appear to be related to the light theme

**Review by amandaejohnston — APPROVED** [review]
